umake/network/download_center.py

```diff
@@ -155,6 +155,8 @@ def _report(block_no, block_size, total_size):
                 actual_checksum = self.md5_for_fd(dest)
             elif checksum_type is ChecksumType.sha256:
                 actual_checksum = self.sha256_for_fd(dest)
+            elif checksum_type is ChecksumType.sha512:
+                actual_checksum = self.sha512_for_fd(dest)
```

Comment:
Mind adding a small tests for this? Look at `tests/small/test_download_center.py`, `TestDownloadCenter` class, `test_download_with_sha256sum` for instance (the files that are tested are in `tests/data/server-content/`.